SWLOR.Game.Server/Service/PerkService/PerkRequirementMustHavePerk.cs

```diff
@@ -0,0 +1,45 @@
+﻿using SWLOR.Game.Server.Entity;
+
+namespace SWLOR.Game.Server.Service.PerkService
+{
+    public class PerkRequirementMustHavePerk: IPerkRequirement
+    {
+        private readonly PerkType _mustHavePerkType;
+        private readonly int _mustHavePerkLevel;
+
+        public PerkRequirementMustHavePerk(PerkType mustHavePerkType, int mustHavePerkLevel = 0)
+        {
+            _mustHavePerkType = mustHavePerkType;
+            _mustHavePerkLevel = mustHavePerkLevel;
+        }
+
+        public string CheckRequirements(uint player)
+        {
+            if (_mustHavePerkType == PerkType.Invalid)
+                return string.Empty;
+
+            var perkDetail = Perk.GetPerkDetails(_mustHavePerkType);
+            var playerId = GetObjectUUID(player);
+            var dbPlayer = DB.Get<Player>(playerId);
+
+            if (!dbPlayer.Perks.ContainsKey(_mustHavePerkType))
+                return $"You must have perk: {perkDetail.Name}";
```

Comment:
Let's make the messaging consistent with the one displayed on line 29. Even if the level is zero, we should display the required rank.